src/components/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import fetchArticles from "../api";
+
+function Articles() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    fetchArticles().then((articlesFromApi) => {
+      setArticles(articlesFromApi);
+    });
+  }, []);
+
+  return (
+    <main>
+      <h2>Articles</h2>
+      <form action="">
+        <label htmlFor="sort">Sort By </label>
+        <select id="sort">
+          <option value="">Option 1</option>
+          <option value="">Option 2</option>
+          <option value="">Option 3</option>
+        </select>
+      </form>
+      <ul>
+        {articles.map((article) => {
+          return (
+            <li key={article.article_id}>
```

Comment:
Having the individual article rendering in here does work, it does open you up to components getting more bloated and more difficult to maintain. I'd recommend breaking this out to an articlesCard component or something along those lines but it's completely up to you if you'd prefer to keep this here. 👍